README.md

```diff
@@ -103,6 +103,3 @@ prefix = "blackjack_crates_io"
 ```
 
 Now instead of `@crates_io_serde//:serde`, use `blackjack_crates_io_serde//:serde`.
-
-# Things that don't work yet (but would gladly accept a PR for)
-* Support for Windows and Mac. For the moment everything assumes your host and target is `x86_64-unknown-linux-gnu`
```

Comment:
Maybe we should leave a note here saying that Mac and Windows support is supported, but untested.

Reply:
https://github.com/SirWindfield/blackjack/pull/1
some more progress 
works on mac

Reply:
I haven't had a lot of time lately for programming-related projects :( Would be nice if @wildarch could review the PR and see if it is more inline with the changes that he requested :smile:

Reply:
I've just reviewed the PR, looks great! 😄